json/recyclarr.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "Recyclarr",
+    "slug": "recyclarr",
+    "categories": [
+        16
```

Comment:
Any particular reason for this category instead of putting it with the other arr apps in media?

Reply:
No reason really. As I said multiple times Autobrr and Recyclarr is helper for arr suite for me, that's why I put it there.

Reply:
Considering how many arr scripts we have now, perhaps we should make an `*Arr Suite` category